changelogs/fragments/6-routeros_facts-multiple-entries-per-iface.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "routeros_facts: Allow multiple addresses and neighbors per interface. This makes ansible_net_neighbors a list instead of a dict. (https://github.com/ansible-collections/community.network/pull/6)"
```

Comment:
```suggestion
- "routeros_facts - allow multiple addresses and neighbors per interface. This makes ``ansible_net_neighbors`` a list instead of a dict (https://github.com/ansible-collections/community.network/pull/6)."
```